pkg/mgrconfig/load.go

```diff
@@ -324,6 +327,41 @@ func (cfg *Config) completeBinaries() error {
 	return nil
 }
 
+func (cfg *Config) completeFocusAreas() error {
+	names := map[string]bool{}
+	seenEmptyFilter := false
+	for i, area := range cfg.Experimental.FocusAreas {
+		if area.Name != "" {
+			if names[area.Name] {
+				return fmt.Errorf("duplicate focus area name: %q", area.Name)
+			}
+			names[area.Name] = true
+		}
+		if area.Weight <= 0 {
```

Comment:
Can 0 weight make sense in some cases (when areas overlap, or with default area)?